control-plane/agents/src/bin/core/controller/policies/rebuild_policies.rs

```diff
@@ -86,7 +80,11 @@ trait Policy {
 }
 
 impl Policy for SystemPerf {
-    fn faulted_child_wait_duration(&self, _nexus: &Nexus) -> Duration {
+    fn faulted_child_wait_duration(&self, nexus: &Nexus) -> Duration {
+        if 1 == online_child_count_check(nexus) {
```

Comment:
1 == online_child_count_check(nexus) isnt that clear. can we make it online_child_count_check(nexus) == 1 ? We would like to check online child count to a number right?